docs/user.md

```diff
@@ -281,6 +281,25 @@ spec:
     s3_force_path_style: true
 ```
 
+## Setting up a standby cluster
+
+Standby clusters are like normal cluster but they are streaming from a remote cluster. Patroni supports it, [read this](https://github.com/zalando/patroni/blob/bd2c54581abb42a7d3a3da551edf0b8732eefd27/docs/replica_bootstrap.rst#standby-cluster) to know more about them.
```

Comment:
This MR builds standby cluster using WAL shipping  for both initial bootstrap and keeping it up to date, right? this is not how Patroni's `standby_cluster` works references here works, which configures a streaming replication.  Maybe mode of operation should be reflected somehow in the spec key name to avoid confusion?

Reply:
Hm, yes this is a bit confusing. Maybe this phrase can be moved somewhere to the end of this paragraph - or maybe just reuse the link for the next mentioning of standby_cluster. In the future we can indeed think about adding a `mode` key for different standby settings.

Reply:
every mode would require own parameters, IMHO a bit of future proofing might help, such as:

```
standby:
  wal_shipping:
     s3_bucket: ....
```

then future streaming replication would look like:

```
stanby:
  streaming:
     host: ...
     slot: ...
```